File: PregnancyAppFrontend/src/dtos/patients/table-user-dto.ts
export interface TableUserDto {
  id: string;
  email: string;
  fullName: string;
  phoneNumber: string;
  birthDate: string;  // DateOnly is represented as string in ISO format (YYYY-MM-DD)
}

export interface TableUsersDto {
  total: number;
  tableUsers: TableUserDto[];
}

File: PregnancyAppBackend/Dtos/Web/Patients/TableUserDto.cs
namespace PregnancyAppBackend.Dtos.Web.Patients;

public class TableUserDto
{
    public Guid Id { get; set; }
    public string Email { get; set; } = null!;
    public string FullName { get; set; } = null!;
    public string PhoneNumber { get; set; } = null!;
    public DateOnly BirthDate { get; set; }
}

public class TableUsersDto
{
    public int Total { get; set; }
    public List<TableUserDto> TableUsers { get; set; }
}